dev_guide/routes.adoc

```diff
@@ -133,10 +134,33 @@ custom certificates. Any non-SNI traffic received on port 443 is handled with
 TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
+endif::[]
 
 Further information on all types of
 xref:../architecture/core_concepts/routes.adoc#secured-routes[TLS termination]
 as well as
 xref:../architecture/core_concepts/routes.adoc#path-based-routes[path-based
 routing] are available in the
 xref:../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[Architecture section].
+
+ifdef::openshift-online[]
+[[custom-route-and-hosts-and-certificates-restrictions]]
+== Restrictions
+
+Currently, custom route hosts are not permitted. Instead, the following host
```

Comment:
There is some nuance beyond what the qualifier "currently" conveys; namely, this restriction will be lifted for accounts with paid plans, but not for unpaid accounts (such as on Developer Preview).  For users with unpaid accounts, the word "currently" may be understood to mean, incorrectly, that the restriction is temporary.